data-raw/stress_test_arguments.R

```diff
@@ -4,7 +4,7 @@ stress_test_arguments <- tibble::tribble(
                "asset_type", "character",       NA, "equity, bonds, loans",         NA,         NA,
         "lgd_senior_claims",    "double",   "0.45",                     NA,      "0.3",      "0.6",
   "lgd_subordinated_claims",    "double",   "0.75",                     NA,      "0.6",      "0.9",
-           "terminal_value",    "double",      "0",                     NA,        "0",      "0.1",
+#           "terminal_value",    "double",      "0",                     NA,        "0",      "0.1",
```

Comment:
Cool. Shall I delete this line? I'll submit a follow up updating the snapshot of this dataset. I could that both at once.

Reply:
i updated the snap I think when prompted to? Or do you see it being out of synch?
I would say the line can stay for now, we will likely bring it back in a few days..

Reply:
Ah, you did well! I thought I was running that test for myself only but obviously not (and it was intentional because it's safe to run for anyone).

A closer look shows this test does not use `is_me()`.

```r
test_that("hasn't changed", {
  expect_snapshot(stress_test_arguments)
})
```